shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/SelectorServiceImpl.java

```diff
@@ -496,6 +496,18 @@ public Boolean enabledByIdsAndNamespaceId(final List<String> ids, final Boolean
             SelectorDO before = JsonUtils.jsonToObject(JsonUtils.toJson(selectorDO), SelectorDO.class);
             selectorDO.setEnabled(enabled);
             if (selectorMapper.updateEnable(id, enabled) > 0) {
+                List<SelectorConditionDO> conditionList = selectorConditionMapper.selectByQuery(new SelectorConditionQuery(selectorDO.getId()));
```

Comment:
why doing this, selectorConditions do not change

Reply:
This is the selector